packages/ui/src/stem-begroot-and-resource-overview/filter/multiselect-tag-filter.tsx

```diff
@@ -36,11 +36,21 @@ const MultiSelectTagFilter = ({
     onlyIncludeIds,
   });
 
+  const randomId = Math.random().toString(36).substring(7);
+
+  function getRandomId(placeholder: string | undefined) {
+    if(placeholder && placeholder.length >= 1) {
+    return placeholder;
```

Comment:
Als je hier een placeholder toevoegt met spaties erin, bijv 'Kies een buurt', dan komt die ook letterlijk in de `htmlFor`, toch? Moet die niet netjes ge-snake-cased (en html safe, dus gekke tekens eruit) worden?